app/scenes/select_server/select_server.js

```diff
@@ -13,21 +13,29 @@ import Button from 'react-native-button';
 import ErrorText from 'app/components/error_text';
 import FormattedText from 'app/components/formatted_text';
 import KeyboardLayout from 'app/components/layout/keyboard_layout';
+import Loading from 'app/components/loading';
 import TextInputWithLocalizedPlaceholder from 'app/components/text_input_with_localized_placeholder';
 import {GlobalStyles} from 'app/styles';
+import {isValidUrl, stripTrailingSlashes} from 'app/utils/url';
 
 import logo from 'assets/images/logo.png';
 
+import {RequestStatus} from 'service/constants';
 import Client from 'service/client';
-import RequestStatus from 'service/constants/request_status';
-
-import {isValidUrl, stripTrailingSlashes} from 'app/utils/url';
 
 export default class SelectServer extends PureComponent {
     static propTypes = {
+        transition: PropTypes.bool.isRequired,
         serverUrl: PropTypes.string.isRequired,
         server: PropTypes.object.isRequired,
-        actions: PropTypes.object.isRequired
+        configRequest: PropTypes.object.isRequired,
+        licenseRequest: PropTypes.object.isRequired,
```

Comment:
Since we've moved these two calls here, do we still get the config and license properly when we already have the server URL configured and go through the Root scene?